spec/hardware/main.fmf

```diff
@@ -19,6 +19,26 @@ description: |
     grouped under a common parent, for example ``cores`` or
     ``model`` under the :ref:`/spec/hardware/cpu` key.
 
+    Operators
```

Comment:
`Value comparison operators` maybe? Rather avoid the generic `Operators` naming?

Reply:
`Operators` seem obvious enough to me...

Reply:
Same here.

Reply:
Let's add "Comparison" - we already have "Logic" operators.

Reply:
Changed in 69169daa.